migrating_from_ocp_3_to_4/migrating-applications-3-4.adoc

```diff
@@ -8,6 +8,16 @@ toc::[]
 
 You can migrate your applications by using the {mtc-full} ({mtc-short}) web console or from the command line.
 
+Most cluster-scoped resources are not yet handled by {mtc-short}. If your applications require cluster-scoped resources, you might have to create them manually on the target cluster.
+
+During migration, the {mtc-full} ({mtc-short}) preserves the following namespace annotations:
```

Comment:
One consideration, you can prolly get away with {mtc-short} like you did in the preceding para. Up to you.